source/carbon/limitations.rst

```diff
@@ -0,0 +1,38 @@
+Limitations
+============
+
+Dates handling
+--------------
```

Comment:
```suggestion
Carbon intensity of electricity
-------------------------------------

The plugin can collect carbon intensity from several sources. The accuracy if these datas varies :
- Data from RTE which represents the carbon intensity of electricity production
- Data from Electricity Map represents the carbon intensity oc electricity consumption (which is more accurate)

Dates handling
--------------
```

The suggestion may be completed with a table if we implement more data sources (Great Britain is in progress)